source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -728,3 +728,23 @@ def _filter_too_much_connections(journeys, instance, request):
             if get_nb_connections(j) > max_connections_allowed:
                 logger.debug("the journey {} has a too much connections, we delete it".format(j.internal_id))
                 mark_as_dead(j, is_debug, "too_much_connections")
+
+
+def remove_excess_tickets_after_culling(response):
+    """
+    Remove excess tickets after culling journeys process
+    """
+    logger = logging.getLogger(__name__)
+
+    fare_ticket_id_list = []
+    for j in response.journeys:
+        fare_ticket_id_list.extend([t_id for t_id in j.fare.ticket_id])
+        # ridesharing case
+        fare_ticket_id_list.extend(
```

Comment:
minor: you may adapt the blind visitor from link-checker (adding just tickets' links) to avoid specific check cases and if we ever add tickets somewhere else in the future https://github.com/CanalTP/navitia/blob/d0c27238dea3d179379a507639f741b81ad397f9/source/jormungandr/tests/check_utils.py#L339-L345

Reply:
I'm lazy... :sleeping: